controllers/mainController.js

```diff
@@ -1,5 +1,48 @@
+var bodyParser = require('body-parser');
```

Comment:
We should probably update this to ES6. I think being consistent will be very helpful down the line